streaming_ws_test.go

```diff
@@ -234,7 +234,7 @@ func TestHandleWS(t *testing.T) {
 }
 
 func TestDialRedirect(t *testing.T) {
-	client := NewClient(&Config{})
+	client := NewClient(&Config{}).NewWSClient()
 	_, err := client.dialRedirect(":")
 	if err == nil {
```

Comment:
Maybe, This is problem of my test code. When func return nil for err expected, it should check `if err != nil` and message should be `should not be fail`. When func return NOT nil for err expected, it should check `if err == nil` and message should be `should be fail`.